tests/main/ubuntu-core-config-defaults-once/task.yaml

```diff
@@ -42,6 +48,11 @@ restore: |
         exit
     fi
 
+    if ! systemctl -l | MATCH "rsyslog.service"; then
```

Comment:
This would prevent the test from running on core18 and checking the ssh system options. Notice, that the test already checks whether the system is core18 and manipulates relevant services only.

Reply:
@bboozzoo thanks for the review, the fix is already pushed